.github/scripts/platform_lib.py

```diff
@@ -385,3 +427,9 @@ def terminate_azure_vms(self, resource_list: List) -> None:
                 print(f"Failed to delete VM {vm['name']}, expected 'None' and got result {deletion_result}")
             else:
                 print(f"Succeeded in deleting VM {vm['name']}")
+
+    def check_and_terminate_select_instances(self, timeout: int, workflow_tag: str) -> None:
```

Comment:
@t14916  These should just throw a not implemented exception (essentially `super`'s implementation, to ensure you get around to implementing them)

Reply:
Yeah I didn't notice these, this should just return the super's not implemented error. I would remove both of these from the azure platform lib.

Reply:
Done